test/src/node.rs

```diff
@@ -94,6 +94,15 @@ impl Node {
         &self.working_dir
     }
 
+    pub fn log_file(&self) -> String {
```

Comment:
node_log helper fn is enough, better to return a Path or PathBuf here if you want to add this helper fn to Node struct